apis/kueue/v1beta1/workload_types.go

```diff
@@ -296,6 +312,19 @@ type WorkloadList struct {
 	Items           []Workload `json:"items"`
 }
 
+// QueueingPolicyType specifies if a suspended job should be queued.
+type QueueingPolicyType string
+
+const (
+	// QueueingPolicyTypeNever means that we don't queue a running workload
+	// after it was suspended
+	QueueingPolicyTypeNever QueueingPolicyType = "Never"
```

Comment:
There is an ongoing discussion about the naming here (https://github.com/kubernetes-sigs/kueue/pull/1252#discussion_r1396325245) about using Hold/Admit as possible values. I'll quote this comment and mention you in that discussion so that everyone is aware.